content/en/docs/tutorials/compose-migration/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Migrating from compose workflows to Kubernetes
+weight: 60
+---
+
+## Disclaimer
+
+The [compose specification](https://github.com/compose-spec/compose-spec) is evolving, this documentation aims to cover the latest version of this compose specification.
+We will assume that we are talking about the latest compose specifications.
+If your compose files at not at the latest version specification, we suggest you try to upgrade to the latest spec version first. 
+
+## Differences between compose specification and Kubernetes specification
+
+A set of workflows defined using the compose specification cannot be automated translated to a unique set of Kubernetes objects.
+While some notions are very similar, there is not a single way to translate concepts from compose to Kubernetes.
+For instance, a [service](https://github.com/compose-spec/compose-spec/blob/master/spec.md#services-top-level-element) concept in compose can be translated in different way in Kubernetes.
+In particular a service in compose **IS NOT** equivalent to a service in Kubernetes.
```

Comment:
This paragraph is very confusing. Could you just say "There is no direct equivalence for many concepts in compose in Kubernetes" instead?